cmake/modules/FindFFTW3.cmake

```diff
@@ -70,6 +70,11 @@ if (FFTW3F_LIBRARY)
 	check_symbol_exists (fftwf_import_wisdom_from_filename fftw3.h FFTW3F_VERSION_RECENT)
 	cmake_pop_check_state() # restore state of CMAKE_REQUIRED_*
 
+	# For some reason in my machine check_symbol_exists() call above is not able to its job
```

Comment:
I don't think it's a good idea to commit your WIN32 trick here, since the Windows CI doesn't have such problem.

Reply:
Hmm, that change slipped in but I actually wanted to submit it as an separate PR. Yes, you are right that Windows CI does not seem to have that problem. But I do, and since it's me who makes the installers, I need to have it working preferably to always have an uncommitted change in my local repo (which the tyrant Git refuses to update)

Reply:
Any way to work around having to commit this into the FindFFTW.cmake file, i.e., via some conditional include statement or similar?

Reply:
I already wrapped it under a ``if (WIN32 AND NOT FFTW3F_VERSION_RECENT)``. According to the comment in line 69, this would only break for the case of someone building on windows with a static FFTW3 build. A case that as far as we know, never happened before, nor is likely to happen in a foreseeable future.